### DISCUSSION TITLE
Applying without full name as GitHub name

### DISCUSSION THREAD

**Discussion by iamk3v** [community/community#64217] [GitHub Education]
I recently applied to the education program, but was rejected because they were unable to verify based on the information i provided. 
I provided a valid email, and a valid semester receipt, with the name of the school and dates of which its valid.

It suggested that i should make my GitHub name my full name, but i rather keep it how it is (Dont like my full name being the front name). Is it possible to be verified without changing name?

**Comment by Jaber-Saed** [comment]
I'm sorry to hear that you had trouble with the education program verification process. While I don't have access to your personal information or the specific details of your application, I can provide some general guidance that might help you navigate this situation:

1. **Contact GitHub Support**: If you believe there was a misunderstanding or error in the verification process, consider reaching out to GitHub Support. They might be able to provide you with more specific guidance and assist you in resolving any issues.

2. **Clarify Verification Details**: Double-check the verification requirements of the education program you applied for. Ensure that you've provided all the necessary information accurately, including a valid email, a valid semester receipt, and any other required documentation.

3. **GitHub Username**: While some programs might suggest using your full name as your GitHub username for verification purposes, it's best to adhere to the program's guidelines. However, if you have concerns about privacy, you can reach out to GitHub Support to discuss alternatives or clarify how you can meet the verification requirements without using your full name as your username.

4. **Privacy Considerations**: If you're uncomfortable with changing your GitHub username, it's important to weigh your privacy preferences against the requirements of the education program. Depending on the program's policies and verification process, they might require certain information to ensure the legitimacy of applications.

5. **Appeal Process**: Some programs might have an appeal process in place if your initial application is rejected. Check the program's documentation or contact their support to inquire about potential appeals.

Remember that communication is key in these situations. If you're unable to find a solution through the program's official channels or guidelines, reaching out to their support team can provide more personalized assistance. Keep in mind that different programs might have different policies and requirements, so it's important to work within their guidelines while also addressing your concerns.

**Comment by ghost** [comment]
man is that comment literally a chatgpt copypaste

**Comment by tuves** [comment]
Hi @Jaber-Saed,

Thanks so much for participating in the Community! While we encourage non-AI answers to questions in community, we also love experimenting and using new technologies. We ask if you do use ChatGPT or similar programs, please disclose what software and model version number you used to generate the response you used in your reply in order to promote transparency in the community.

ChatGPT answers in a tone that sounds like a tech support professional and while we support trying out this software, it is important to verify that they are solutions before posting otherwise it may mislead users into thinking that they are receiving official responses when they are not.

Per ChatGPT's website, the program does have limitations like:
- "May occasionally generate incorrect information"
- "May occasionally produce harmful instructions"
- "Limited knowledge of world and events after 2021"
